graylog2-server/src/main/java/org/graylog2/security/ldap/LdapConnector.java

```diff
@@ -326,7 +326,7 @@ private String normalizedDn(String dn) {
             try {
                 return new Dn(dn).getNormName();
             } catch (LdapInvalidDnException e) {
-                LOG.warn("Invalid DN", e);
+                LOG.debug("Invalid DN", e);
```

Comment:
I'm totally for this change but I fear that the handling inside the `catch` block will leave people clueless why stuff didn't work.

The caller of this method has basically no idea whether he really got back a normalized DN or an invalid DN. 😞